evalml/pipelines/regression/timeseries_baseline_regression.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+
+from evalml.pipelines import TimeSeriesRegressionPipeline
+from evalml.utils.gen_utils import (
+    _convert_to_woodwork_structure,
+    _convert_woodwork_types_wrapper,
+    pad_with_nans
+)
+
+
+class TimeSeriesBaselineRegressionPipeline(TimeSeriesRegressionPipeline):
+    """Baseline Pipeline for time series regression problems."""
+    _name = "Time Series Baseline Regression Pipeline"
+    component_graph = ["Time Series Baseline Regressor"]
+
+    def predict(self, X, y=None, objective=None):
+        """Make predictions using selected features.
+
+        Arguments:
+            X (ww.DataTable, pd.DataFrame, or np.ndarray): Data of shape [n_samples, n_features]
+            y (ww.DataColumn, pd.Series, np.ndarray, None): The target training targets of length [n_samples]
```

Comment:
not-blocking: Maybe we shouldn't set `y` to `None` by default and update the docstring since `None` isn't allowed?